cmake/common.cmake

```diff
@@ -748,3 +748,49 @@ function(add_picobench name)
 
   set_property(TEST ${name} PROPERTY LABELS benchmark)
 endfunction()
+
+# Perf tool executable
+set(SUBMITTER_DIR ${CCF_DIR}/tests/perf-system/submitter)
+
+add_executable(
+  submit ${SUBMITTER_DIR}/submit.cpp ${SUBMITTER_DIR}/handle_arguments.hpp
+         ${SUBMITTER_DIR}/parquet_data.hpp
+)
```

Comment:
I don't think this should live in `common.cmake`. Ideally it should go in `tests/.../submitter/CMakeLists.txt`, and then that should be included from the root `CMakeLists.txt` or similar.

Reply:
I made a new CMakeLists.txt inside submitter directory and I included it in  CMakeLists.txt as `include(${CCF_DIR}/tests/perf-system/submitter/CMakeLists.txt)`.
If you think it looks good, please feel free to resolve the conversation. Thanks!